velox/type/Variant.h

```diff
@@ -153,6 +153,51 @@ struct DecimalCapsule {
     return lhsIntegral < rhsIntegral;
   }
 
+  bool operator<=(const DecimalCapsule& other) const {
```

Comment:
Could we implement these operators by calling the < and == operators above ? It would be better for code reuse. I think in C++ its possible for one operator to call another.

Reply:
Thanks for the suggestion, updated.